spec/features/book_index_redirect_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.feature 'Book index redirect', type: :feature do
+  before(:each) do
+    create(:book)
+    Book.reindex
+
+    visit root_path
+  end
+
+  context 'user search:' do
+    it 'specific world' do
+      fill_in 'Book title', with: 'Yard'
```

Comment:
add only your test expectations to `it` statement and move everything else to before block or let method helper